Classes/Core/BasicRequirements.php

```diff
@@ -165,7 +163,29 @@ protected function checkFilePermissions()
                 return new Error('The folder "%s" is not writable. Check your file permissions (did you use flow:core:setfilepermissions?)', 1330372964, [$folderPath]);
             }
         }
+        if (!$this->checkFileGroupWritePermission()) {
+            return new Error('Files are not created with group write permissions. Check your webserver setup. On systems without filesystem ACLs, you need to set umask 0002 for the PHP/webserver daemon.');
+        }
+    }
 
-        return null;
+    /**
+     * Check that files are being created with group write permissions.
+     *
+     * @return bool
+     */
+    protected function checkFileGroupWritePermission()
+    {
+        $testfolder = FLOW_PATH_ROOT . 'Test';
```

Comment:
Actually, is this required to be writable? On a "really secure system" I'd only make `Web/_Resources`, `Configuratio` (_sigh_) and `Data` writable, as that should suffice. So maybe test in `FLOW_PATH_DATA`?

Reply:
`Configuration` no longer needed YAY